test/sassc_rails_test.rb

```diff
@@ -243,6 +245,18 @@ def test_allows_for_inclusion_of_inline_source_maps
     assert_match /sourceMappingURL/, asset
   end
 
+  def test_adds_source_map_in_debug_mode
+    if Sprockets::VERSION.start_with?("4")
```

Comment:
I think you should definitely write the test so if sprockets 5 is ever released, the test will be run under sprockets 5 too and raise on failure, instead of just skipping this test!

Reply:
I changed to make this working with Sprockets >= 4